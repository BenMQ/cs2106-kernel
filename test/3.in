cr a 2
cr b 1
cr c 2
to
cr d 2
req R1 1
to
cr e 1
req R1 1
req R3 3
cr f 1
to
req R2 2
req R3 1
cr g 1
de f
req R1 1
req R4 3
to
de d
