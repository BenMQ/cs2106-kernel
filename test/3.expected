init a a a c c c a a d d d c c d d d b b e error
